### ISSUE TITLE
Google Home Cannot Discover Bridge

### ISSUE THREAD

**Issue by JordanBuser** [bwssytems/ha-bridge#329]
Hi,

I am running ha-bridge on an old raspberry pi, am running 3.5.1 on port 80 without any issues that I can discern.  The web UI shows all my Vera Edge devices and allows me to turn them on and off.  When I go to the Home app on my phone and attempt to add Philips Hue devices, I get redirected to the Hue login page.  Have read that others had this problem and suggested to ensure nothing else on my network was using port 80, which I have done but the Home app still cannot find the bridge.

I've enabled trace UPNP and below are some logs from when I was trying to add lights via the app.  My ha-bridge is on 192.168.1.97, my Vera Edge is 192.168.1.89 and my Google home is 192.168.1.80.  Thanks!

12-26-2016 23:45:03.105	INFO	Traceupnp: isSSDPDiscovery found message to be an M-SEARCH message.	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:03.111	INFO	Traceupnp: isSSDPDiscovery found message to be valid under strict rules - strict: true	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:03.112	INFO	Traceupnp: SSDP packet from 192.168.1.80:51415, body: M-SEARCH * HTTP/1.1
 HOST: 239.255.255.250:1900
 MAN: "ssdp:discover"
 MX: 2
 ST: ssdp:all
 
	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:03.117	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate1 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: upnp:rootdevice
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336::upnp:rootdevice
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:03.119	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate2 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:03.121	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate3 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: urn:schemas-upnp-org:device:basic:1
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.264	INFO	Traceupnp: isSSDPDiscovery found message to be an M-SEARCH message.	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.265	INFO	Traceupnp: isSSDPDiscovery found message to be valid under strict rules - strict: true	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.266	INFO	Traceupnp: SSDP packet from 192.168.1.108:13954, body: M-SEARCH * HTTP/1.1
 MX: 5
 ST: upnp:rootdevice
 MAN: "ssdp:discover"
 User-Agent: UPnP/1.0 DLNADOC/1.50 Kodi
 Connection: close
 Host: 239.255.255.250:1900
 
	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.271	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate1 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: upnp:rootdevice
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336::upnp:rootdevice
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.273	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate2 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.279	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate3 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: urn:schemas-upnp-org:device:basic:1
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.290	INFO	Traceupnp: isSSDPDiscovery found message to be an M-SEARCH message.	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.291	INFO	Traceupnp: isSSDPDiscovery found message to be valid under strict rules - strict: true	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.292	INFO	Traceupnp: SSDP packet from 192.168.1.108:13954, body: M-SEARCH * HTTP/1.1
 MX: 5
 ST: upnp:rootdevice
 MAN: "ssdp:discover"
 User-Agent: UPnP/1.0 DLNADOC/1.50 Kodi
 Connection: close
 Host: 239.255.255.250:1900
 
	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.320	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate1 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: upnp:rootdevice
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336::upnp:rootdevice
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.322	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate2 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener
12-26-2016 23:45:12.324	INFO	Traceupnp: sendUpnpResponse discovery responseTemplate3 is <<<HTTP/1.1 200 OK
 HOST: 239.255.255.250:1900
 CACHE-CONTROL: max-age=100
 EXT:
 LOCATION: http://192.168.1.97:80/description.xml
 SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.15.0
 hue-bridgeid: B827EBFFFEA54336
 ST: urn:schemas-upnp-org:device:basic:1
 USN: uuid:2f402f80-da50-11e1-9b23-b827eba54336
 
 >>>	com.bwssystems.HABridge.upnp.UpnpListener

**Comment by bwssytems** [comment]
Search in the closed issues, this has been discussed.

**Comment by adamdiel** [comment]
For what it's worth I'm also having this issue. When I first set it up yesterday, I got it to discover the bridge no problem and had Google Home talking to my devices. Today Google is saying the devices were unavailable. 

The IP address of the Raspberry Pi where the bridge is running hasn't changed, I can still access the bridge and turn the lights on and off from there. 

One thing that came to mind was the fact I have a 5GHz network (Google Home) and a 2.4GHz network (Raspberry Pi) both coming from the same router. 

I tried to moving my Google Home to the 2.4GHz network and had no luck even after a reboot of both the google home and ha-bridge. 

Here is a gist of my log if it helps. My ha-bridge is 192.168.1.48 and the google home is 192.168.1.47
https://gist.github.com/adamdiel/53edb3c406b67c6607dc464fb5699b86

Tomorrow I will try moving to the Raspberry Pi to a wired network to see if that changes anything. 

Also if it helps, I'm running the latest Rasbian Jesse release.
